doc/src/Howto_CAC.txt

```diff
@@ -0,0 +1,245 @@
+"Higher level section"_Howto.html - "LAMMPS WWW Site"_lws - "LAMMPS
+Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Commands_all.html)
+
+:line
+
+CAC method :h3
+
+[Overview:]
+
+The Concurrent Atomistic Continuum (CAC) method, as implemented in LAMMPS,
+is a concurrent multiscale method implemented using the Finite Element 
+Method (FEM) that consistently solves systems with 
+both coarsely-meshed, using finite elements, and atomistic regions. This is achieved by defining
+a non-local force density at all finite elements, as well as the typical particle force
+at atoms, using the interatomic potentials that are ubiquitous in Molecular Dynamics. The theoretical
+framework that makes this possible is an atomistic formulation of the field representation
+of conservation laws; this formulation is constructed around the concepts of discontinuous atomic
+level structures and non-local atomic interaction as presented in "(Chen)"_#USER-CAC-Chen
```

Comment:
I'm guessing that this is the line being flagged by the doc build warnings but I'm confused as to the why.